Radzen.Blazor/RadzenFormField.razor

```diff
@@ -1,32 +1,32 @@
-@using Radzen.Blazor
-@inherits RadzenComponent
-
-@if (Visible)
-{
-<div @ref="@Element" for="@Component" style="@Style" @attributes="Attributes" class="@GetCssClass()" id="@GetId()" onfocusin="this.classList.add('rz-state-focused')" onfocusout="this.classList.remove('rz-state-focused')">
-    <div class="rz-form-field-content">
-        <CascadingValue Value=@context>
-        @if (Start != null)
-        {
-        <div class="rz-form-field-start">
-            @Start
-        </div>
-        }
-        @ChildContent
-        <label class="rz-form-field-label rz-text-truncate" for=@Component>@Text</label>
-        @if (End != null)
-        {
-        <div class="rz-form-field-end">
-            @End
-        </div>
-        }
-        </CascadingValue>
-    </div>
-    @if (Helper != null)
-    {
-    <div class="rz-form-field-helper">
-        @Helper
-    </div>
-    }
-</div>
-}
+@using Radzen.Blazor
+@inherits RadzenComponent
+
+@if (Visible)
+{
+<div @ref="@Element" for="@Component" style="@Style" @attributes="Attributes" class="@GetCssClass()" id="@GetId()" onfocusin="this.classList.add('rz-state-focused')" onfocusout="this.classList.remove('rz-state-focused')">
+    <div class="rz-form-field-content">
+        <CascadingValue Value=@context>
+        @if (Start != null)
+        {
+        <div class="rz-form-field-start">
+            @Start
+        </div>
+        }
+        @ChildContent
+        <label class="rz-form-field-label rz-text-truncate" for=@Component>@if (TextTemplate != null) {@Text} else {@TextTemplate}</label>
+        @if (End != null)
+        {
+        <div class="rz-form-field-end">
+            @End
+        </div>
+        }
+        </CascadingValue>
+    </div>
+    @if (Helper != null)
+    {
+    <div class="rz-form-field-helper">
+        @Helper
+    </div>
+    }
+</div>
+}

```

Comment:
You have changed the formatting of the whole file. Please modify just the line that renders the text.